ts/navigation/MainNavigator.tsx

```diff
@@ -56,9 +56,7 @@ const styles = StyleSheet.create({
   labelStyle: {
     ...makeFontStyleObject(Platform.select),
     marginBottom: 7,
-    width: 64,
```

Comment:
it's not clear to me why we nned to align the text to the icons using fixed position ?
I guess that icons and text are simply centered in their of cell (column).
probably a text-align: center or using flex justify will do the trick

Reply:
That's why I'm getting rid of `width` and `marginRight` :) I first based this too closely on the invision and may have inadvertently introduced these fixed values (which worked nicely on the iPhone since it has the same proportions as the invision)